### ISSUE TITLE
Published to npm?

### ISSUE THREAD

**Issue by chmac** [denorg/recursive-readdir#2]
I found this package on the npm registry. Yet it relies on all the `Deno` namespace operators. Maybe it was published by accident?

https://www.npmjs.com/package/@denorg/recursive-readdir

**Comment by AnandChowdhary** [comment]
Yep, we publish our packages to npm to get the free CDNs that come with it :)

**Comment by chmac** [comment]
Hahaha, makes sense! Maybe it's worth putting a note in the README that gets published to npm to say that it's not a node compatible package?

I was looking for a recursive readdir package that would work with both npm and deno and was excited to find this in npm until I realised that it won't actually run under node!
